docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+export ZITI_PKI_EDGE_CERT="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/certs/${ZITI_EDGE_API_PKI_NAME}-client.cert"
+export ZITI_PKI_EDGE_CA="${ZITI_PKI}/${ZITI_EDGE_CA_NAME}/edge.cas.pem"
+
+export ZITI_PKI_SIGNER_KEY="${ZITI_PKI}/${ZITI_SIGN_CA_NAME}/keys/${ZITI_SIGN_CA_NAME}.key"
+export ZITI_PKI_SIGNER_CERT="${ZITI_PKI}/${ZITI_SIGN_CA_NAME}/certs/${ZITI_SIGN_CA_NAME}.chain.pem"
+```
+
+### Create the Controller Config File
+
+The controller config file is populated based on the values of environment variables set up to this point.
+
+```
+"${ZITI_BIN_DIR}/ziti" create config controller >${ZITI_HOME}/${ZITI_HOSTNAME}.yaml
+```
+
+### Initialize the Controller
+
+Initializing the controller sets up database files and some of the configuration values.
+
+```
+mkdir ${ZITI_HOME}/db
+"${ZITI_BIN_DIR}/ziti" controller edge init "${ZITI_HOME}/${ZITI_HOSTNAME}.yaml" -u "admin" -p $ZITI_PWD
+```
+
+### Run the Controller
+
+```
+"${ZITI_BIN_DIR}/ziti" controller run ${ZITI_HOME}/${ZITI_HOSTNAME}.yaml &> ${ZITI_HOME}/${ZITI_HOSTNAME}.log &
+```
+
+### Wait for the Controller
+
+We want to ensure the controller is ready before creating a router, this mini loop
+
+```
+while [[ "$(curl -w "%{http_code}" -m 1 -s -k -o /dev/null https://${ZITI_CTRL_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}/edge/client/v1/version)" != "200" ]]; do
+  echo "waiting for https://${ZITI_CTRL_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}"
+  sleep 1
+done
+```
+
+## Create Router
+
+### Create the Router Entity
+
+The router needs to be created through the controller. This will generate a one-time token to be used during enrollment.
+
+```
+# We have to log in first
+"${ZITI_BIN_DIR}/ziti" edge login ${ZITI_CTRL_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT} -u admin -p $ZITI_PWD -y
+
+"${ZITI_BIN_DIR}/ziti" edge create edge-router ${ZITI_HOSTNAME}-edge-router -o ${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.jwt -t -a public
+```
+
+### Create the Router Config File
+
+Just as with the controller, we need to create a router config file. The router config also uses values set in environment variables up to this point.
+
+```
+"${ZITI_BIN_DIR}/ziti" create config router edge --routerName ${ZITI_HOSTNAME}-edge-router >${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.yaml
+```
+
+### Enroll the Router with the Controller
+
+Enroll the router with the controller utilizing the config file and enrollment token previously generated.
+
+```
+"${ZITI_BIN_DIR}/ziti" router enroll ${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.yaml --jwt ${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.jwt
+```
+
+### Run the Router
+
+```
+"${ZITI_BIN_DIR}/ziti" router run "${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.yaml" &> ${ZITI_HOME}/${ZITI_HOSTNAME}-edge-router.log &
+```
+
+## Confirm the Network is Up
+
+Log in and run a command to list the edge routers and we should see a single edge router showing `ONLINE`.
```

Comment:
the quickstat does not leave the network in a state that this can be done, so technically this is inaccurate. you'd have to startController and startRouter but the quickstart doesn't do that. I'd say the ending is when the 'success' banner is printed